### ISSUE TITLE
Can you make the video publicly accessible?

### ISSUE THREAD

**Issue by jason-huanghao** [amit-sharma/causal-inference-tutorial#7]
Hello, I am very interesting in this tutorial, could you make the video publicly accessible?

**Comment by amit-sharma** [comment]
Yes, it was publicly accessible but there has been some settings change at the Kellogg server.  let me ask them

**Comment by username-rp** [comment]
Just wondering if there has been any updates on this? Would love to watch the video but it is not accessible

**Comment by amit-sharma** [comment]
working on it, will post an update here as soon as it is fixed.

**Comment by ap-2021** [comment]
Really interested...when will the video be publicly available?
